src/plugins/legacy/itkDataSHImage/interactors/itkDataSHImageVtkViewInteractor.cpp

```diff
@@ -263,7 +263,6 @@ void itkDataSHImageVtkViewInteractor::setupParameters()
     medBoolParameterL *flipXParam = new medBoolParameterL("FlipX", this);
     medBoolParameterL *flipYParam = new medBoolParameterL("FlipY", this);
     medBoolParameterL *flipZParam = new medBoolParameterL("FlipZ", this);
-    flipZParam->setValue(true);
```

Comment:
My PR on SH changes was not applied to 3.1.x ? It's weird these modifications appear in this PR